calculator.js

```diff
@@ -1,19 +1,67 @@
+// const prompt = require('prompt')
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+
+// const exampleAdditionInput = {
+//   num1: 3,
+//   num2: 5,
+//   operation: 'add',
+// }
+
+const add = function(num1, num2) {
+  console.log(`${num1} + ${num2} = ${Number(num1) + Number(num2)}`)
+}
+
+const subtract = function(num1, num2) {
+  console.log(`${num1} - ${num2} = ${Number(num1) - Number(num2)}`);
+}
+
+const multiply = function(num1, num2) {
+  console.log(`${num1} * ${num2} = ${Number(num1) * Number(num2)}`);
+}
+
+const divide = function(num1, num2) {
+  if (num2 == 0) {
+    console.log('You cannot divide by zero')
+  } else {
+    console.log(`${num1} / ${num2} = ${Number(num1) / Number(num2)}`) 
+  }
+  
 }
 
 const calculateUserInput = function (error, promptInput) {
   console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
```

Comment:
Not a big deal but you were expected to remove this logging.  It was just to get started.